feature/metis/manage-conversations/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/feature/metis/manageconversations/ui/conversation/settings/PerformActionOnUserDialogs.kt

```diff
@@ -95,10 +96,10 @@ private fun PerformActionOnUserDialogs(
                     onDismissRequest = onDismiss
                 )
 
-                is GroupChat -> KickUserFromChannelDialog(
+                is GroupChat -> KickUserFromGroupDialog (
                     humanReadableName = humanReadableName,
                     username = username,
-                    channelName = conversation.humanReadableTitle,
+                    groupName = conversation.humanReadableTitle,
```

Comment:
The `groupName` is not used anymore in the `KickUserFromGroupDialog`. Either remove the parameter or include it in the dialog texts